web/v2/index.ts

```diff
@@ -126,7 +126,7 @@ export const APIWebRoutes = {
    *
    * @returns APIResponse<APICompanyRoles>
    */
-  companyRoles(companyId: string | number) {
+  companyRoles(companyId: number) {
```

Comment:
This should be consistent with the others including companyId and accept a string as well.